ReleaseNotes-2.10.19

```diff
@@ -0,0 +1,20 @@
+#
+# Open Build Service 2.10.19
+#
+
+Please read the README.md file for initial installation
+instructions or use the OBS Appliance from
+
+  http://openbuildservice.org/download/
+
+The dist/README.UPDATERS file has information for people updating
+from a previous OBS release.
+
+
+Bugfixes
+========
+Frontend:
+ * Update rails-html-sanitizer to 1.4.4
+   - CVE-2022-32209 Rails::Html::Sanitizer vulnerable to Cross-site Scripting 
+ *  fixup! [backend] Add support for qemu system emulated builds via bs_worker
```

Comment:
ReleaseNotes are for admins, not for developers. You will have to turn this into an admin understandable sentence.